main.go

```diff
@@ -261,6 +263,10 @@ func main() {
 	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "UserSignupCleanup")
 	}
+
+	if err = space.SetupWithManager(mgr, namespace, memberClusters); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "Space")
+	}
```

Comment:
out of curiosity - why do you use a different way of creating the controller compared to the other controllers? The previous format is taken from the code that was generated by operator-sdk - I thought that we wanted to be as close to it as possible.

Reply:
refactored in 6b4027f